src/dashboard/src/components/ingest/views.py

```diff
@@ -494,10 +564,34 @@ def transfer_backlog(request):
         if models.SIPArrange.objects.filter(
             original_path__endswith=path['relative_path']).exists():
             not_draggable = True
-        _es_results_to_directory_tree(path['relative_path'], return_list, not_draggable=not_draggable)
+        if ui == 'legacy':
+            _es_results_to_directory_tree(path['relative_path'], return_list, not_draggable=not_draggable)
+            response = return_list
+        else:
+            _es_results_to_appraisal_tab_format(path, directory_map, return_list, not_draggable=not_draggable)
+            response = {
+                "formats": [],  # TODO populate this
```

Comment:
It is, yeah; not having this in right now is making testing future PRs a bit harder so I'd like to get it in sooner.